app/Http/Controllers/CelebController.php

```diff
@@ -22,14 +23,16 @@ public function create()
         return view('celebs/create');
     }
 
-    public function store(Request $request)
+    public function store(CelebRequest $request)
     {
-        //
+        Celeb::create($request->validated());
+
+        return redirect('celebs');
     }
 
     public function show(Celeb $celeb)
     {
-        $celeb->with('movies')->get();
+        $celeb->with('movies')->paginate(5);
```

Comment:
Hey Kieran,

Nice idea to paginate the movies. However, if you check the view it does not seem to be working as expected. 
Try to look around for options to paginate relationships in Laravel, you might need to store this in another variable :D and call `movies()` instead of `with`. 

![Screen Shot 2021-04-29 at 10 49 49 am](https://user-images.githubusercontent.com/53244381/116489572-a172f500-a8d8-11eb-8afe-0dbf059f7937.png)